corporate/views.py

```diff
@@ -417,12 +417,7 @@ def update_plan(
                     licenses=licenses
                 )
             )
-        try:
-            validate_licenses(
-                plan.charge_automatically, licenses, get_latest_seat_count(user.realm)
-            )
-        except BillingError as e:
-            return json_error(e.message, data={"error_description": e.description})
+        validate_licenses(plan.charge_automatically, licenses, get_latest_seat_count(user.realm))
```

Comment:
Fix the commit message typo and documented the benefits of removing these exception handlers and then merged this as 72f9f964a1e18c444d593ca4f86fdf73a7309867.